vbeam/apodization/plotting.py

```diff
@@ -43,16 +43,16 @@ def plot_apodization(
 
     # Calculate apodization values
     vals = get_apodization_values(
-        apodization,
-        sender,
-        point_position,
-        receiver,
-        wave_data,
-        spec,
+        apodization=apodization,
+        sender=sender,
+        point_position=point_position,
+        receiver=receiver,
+        wave_data=wave_data,
+        spec=spec,
         # We want to keep the dimensions of point_position
-        spec["point_position"].tree,
-        average_overlap,
-        jit,
+        dimensions=spec["point_position"].tree,
```

Comment:
I was not 100% certain whether it should be
`dimensions=spec["point_position"].tree` or `reduce_sum_dimensions=spec["point_position"].tree`, but this seems to work